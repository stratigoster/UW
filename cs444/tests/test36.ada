PACKAGE foo IS
  function plus (a,b:integer) return integer;
  function myplus () return integer;
--                ^^
-- you don't need brackets here if there are no parameters
END foo;
