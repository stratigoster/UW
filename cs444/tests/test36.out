***ERROR***:3:20:	Expected "Id" but found ")"
